apis/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
is it necessary, AFAIK license should be at the project's root directory.

Reply:
It's because Knative fetches only this module and looks for the LICENSE file. Since we are exporting only this module, it makes sense having a copy of the license here as well.

Reply:
I should've added a disclaimer in the PR description.